src/engine/renderer/tr_public.h

```diff
@@ -59,6 +59,9 @@ struct glconfig2_t
 	bool glCoreProfile;
 	bool glForwardCompatibleContext;
 
+	glDriverVendor_t driverVendor;
+	glHardwareVendor_t hardwareVendor;
```

Comment:
These values will be undefined if no driver/hardware is detected, which will likely lead to a crash.

Reply:
In `RE_BeginRegistration()` we already do `*glconfig2 = glconfig2_t{};` and `0` will just be considered as `UNKNOWN`, but I may do things in more explicit way.

Reply:
I see, missed that one.

Reply:
Done, they are explicitly reset at the beginning of the detection function, and I added some asserts into the print functions.